pandasai/llm/base.py

```diff
@@ -375,7 +375,7 @@ class BaseGoogle(LLM):
 
     temperature: Optional[float] = 0
     top_p: Optional[float] = 0.8
-    top_k: Optional[float] = 0.3
+    top_k: Optional[int] = 40
```

Comment:
What if we rather just make `top_p` and `top_k` default to `None`?

Reply:
@gventuri  User might forgot to set them so the default values are good for it .....